core/example/tutorial/14_unmanaged_view/parallel_for_example.cpp

```diff
@@ -0,0 +1,135 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+#include <Cabana_Sort.hpp>
+
+#include <cstdio>
+
+//---------------------------------------------------------------------------//
+// parallel for using an unmanaged view.
+//---------------------------------------------------------------------------//
+void parallelForExample()
+{
+    /* Declare general run parameters */
```

Comment:
Can you add a general description comment here at the beginning?

Reply:
I thought about this... but I really didn't know what to say. @sslattery do you have a good feel of what to say about this ?

Reply:
I would describe the premise of an unmanaged `AoSoA` and examples of why one would want to use it. The main point is someone has allocated data elsewhere that is directly mappable to a `Cabana::AoSoA` and that person would like to use Cabana algorithms in their codes. I would also comment on how you composed the user C struct and why the data layout maps to an AoSoA. Adding the extra dimension to `a` was a good idea here because it exposes the user to how the inner array layouts must be setup - with the coalesced index defined last.

Reply:
Added an attempt to address this